raiden/tests/scenarios/mfee3_only_imbalance_fees.yaml

```diff
@@ -63,11 +64,12 @@ scenario:
                 expected_routes:
                   - [0, 1, 2]
 
-            # -2000 TKN fees are calculated, no margin is added for negative fee values
+            # -2000 TKN fees are calculated, but the actual value is close to zero because
+            # 1. Due to the negative fee less tokens are transferred, resulting in a smaller negative fee
```

Comment:
There are no more negative fees, right?

Reply:
There are, when capping of fees is disabled. This is done in https://github.com/raiden-network/raiden/pull/5139/files#diff-e6f7856d552ed70edb56ef4bcd1ca822R39